### PR TITLE
Add new measurement family in concepts

### PR DESCRIPTION
**Pull request by Delphineray**

### CODE REVIEW

File: content/concepts/target-market-settings.md [hunk 1]
```diff
@@ -122,6 +128,9 @@ Below is the JSON standard format representing this set of channels when request
 :::
 
 ## Currency _(v2+)_
```
Comment (NolwennP) [inline content/concepts/target-market-settings.md]:
```suggestion
## Currency
```
Thanks Delph!

File: content/concepts/target-market-settings.md [hunk 2]
```diff
@@ -171,9 +187,64 @@ Below is the JSON standard format representing a measure family.
...
+::: panel-link Want more details about the measure family resource? [Check its endpoints here!](/api-reference.html#Measurefamily)
+:::
+
+## Measurement family
+::: availability versions=Serenity editions=CE,EE
+:::
+
+If you want to store measurement regarding your product such as weight, height or power inside your PIM, you will need measurement families. These entities will be really helpful in the case you are requesting products for a given channel and you want these measurement attributes to be converted into the units you specified in your channel.
+
+Below is an example of one of these measurement attribute.
```
Comment (NolwennP) [inline content/concepts/target-market-settings.md]:
```suggestion
Below is an example of one of these measurement attributes.
```

File: content/concepts/target-market-settings.md [hunk 3]
```diff
@@ -171,9 +187,64 @@ Below is the JSON standard format representing a measure family.
...
+Below is the JSON standard format representing a measurement family.
+
+```json
+{
+  "code": "AREA",
+  "labels": {
+    "en_US": "Area",
+    "fr_FR": "Surface"
+  },
+  "standard_unit_code": "SQUARE_METER",
```
Comment (NolwennP) [inline content/concepts/target-market-settings.md]: I would add in the `units` array below this unit to be extra clear.

File: content/concepts/target-market-settings.md [hunk 4]
```diff
@@ -171,9 +187,64 @@ Below is the JSON standard format representing a measure family.
...
+      "symbol": "cm²"
+    },...
+  ]
+}
+```
+
 ::: warning
-Endpoints for the measure families are only available starting the 2.0 version.
+New endpoints for the measurement families are only available for the Serenity version.
 :::
```
Comment (NolwennP) [inline content/concepts/target-market-settings.md]: To remove.

File: content/concepts/target-market-settings.md [hunk 5]
```diff
@@ -144,12 +153,19 @@ Endpoints for the currencies are only available starting the 2.0 version.
 ::: panel-link Want more details about the currency resource? [Check its endpoints here!](/api-reference.html#Currency)
 :::
 
-## Measure family _(v2+)_
+## Measure family
+::: availability versions=2.x,3.x,4.0,Serenity editions=CE,EE
+:::
+
+::: warning
+For the Serenity version, we encourage you to use [these new endpoints](#measurement-family), as they are more powerful. They allow to create/update measurement families and they guarantee the order of the conversion operations.
```
Comment (camillebourjade) [inline content/concepts/target-market-settings.md]:
```suggestion
For the Serenity version, we encourage you to use [these new endpoints](#measurement-family), as they are more powerful. They allow you to create/update measurement families and they guarantee the order of the conversion operations.
```

File: content/concepts/target-market-settings.md [hunk 6]
```diff
@@ -144,12 +153,19 @@ Endpoints for the currencies are only available starting the 2.0 version.
...
+:::
+
 If you want to store metrics regarding your product such as weight, height or power inside your PIM, you will need measure families. These entities will be really helpful in the case you are requesting products for a given channel and you want these metrics attributes to be converted into the units you specified in your channel.
```
Comment (camillebourjade) [inline content/concepts/target-market-settings.md]:
```suggestion
If you want to store your product metrics, such as weight, height or power, inside your PIM, you will need measure 
families. These entities will be really helpful when you are requesting products for a given channel and you want those metrics attributes to be converted into the units you set in the given channel.
```

File: content/concepts/target-market-settings.md [hunk 7]
```diff
@@ -144,12 +153,19 @@ Endpoints for the currencies are only available starting the 2.0 version.
...
 
 Below is an example of one of these metrics attributes.
```
Comment (camillebourjade) [inline content/concepts/target-market-settings.md]:
```suggestion
Below is an example of one of the metrics attributes.
```

File: content/concepts/target-market-settings.md [hunk 8]
```diff
@@ -171,9 +187,64 @@ Below is the JSON standard format representing a measure family.
...
 ```
 
+::: panel-link Want more details about the measure family resource? [Check its endpoints here!](/api-reference.html#Measurefamily)
+:::
+
+## Measurement family
+::: availability versions=Serenity editions=CE,EE
+:::
+
+If you want to store measurement regarding your product such as weight, height or power inside your PIM, you will need measurement families. These entities will be really helpful in the case you are requesting products for a given channel and you want these measurement attributes to be converted into the units you specified in your channel.
```
Comment (camillebourjade) [inline content/concepts/target-market-settings.md]:
```suggestion
If you want to store your product measurement, i.e. weight, height or power inside your PIM, you will need measurement families. These entities will be really helpful when you are requesting products for a given channel and you want these measurement attributes to be converted into the units you set in the given channel.
```

File: content/concepts/target-market-settings.md [hunk 9]
```diff
@@ -40,6 +43,9 @@ Below is the JSON standard format representing this set of locales.
 :::
 
 ## Channel
+::: availability versions=1.7,2.x,3.x,4.0,Serenity editions=CE,EE
+:::
+
 A channel refers to a place where your product information will be visible: for example, a website, a print catalog or a mobile application.
 Actually, a channel defines a selection of products and information to export.
```
[the remark this answers is not in the document]
Comment (Delphineray) [reply content/concepts/target-market-settings.md]: Not linked to my update

File: content/concepts/target-market-settings.md (hunk 6 above)
Comment (Delphineray) [reply content/concepts/target-market-settings.md]: Not linked to my update

File: content/concepts/target-market-settings.md (hunk 3 above)
Comment (Delphineray) [reply content/concepts/target-market-settings.md]: Ah yes, it's added

File: content/concepts/target-market-settings.md (hunk 4 above)
Comment (Delphineray) [reply content/concepts/target-market-settings.md]: Yes, done

File: content/concepts/target-market-settings.md [hunk 10]
```diff
@@ -171,9 +187,106 @@ Below is the JSON standard format representing a measure family.
...
+You can have at max 100 measurements families and 50 units per measurement family.
+As a consequence, when you ask for the list of measurement families with their units, you'll see that the response is not paginated. It won't cause any performance issue, since you can't have more than 100 measurement families.
+:::
+
+::: panel-link Want more details about the measurement family resource? [Check its endpoints here!](/api-reference.html#Measurementfamily)
+:::
+
+### Focus on the units
+
+For each measurement family, a unit is defined as standard and used to convert the other units using on or several conversion operations.
```
Comment (NolwennP) [inline content/concepts/target-market-settings.md]:
```suggestion
For each measurement family, a unit is defined as standard and used to convert the other units using one or several conversion operations.
```

File: content/concepts/target-market-settings.md [hunk 11]
```diff
@@ -171,9 +187,106 @@ Below is the JSON standard format representing a measure family.
...
+      "operator": "CONVERSION_OPERATOR",
+      "value": "CONVERSION_VALUE"
+    }
+  ],
+  "symbol": "UNIT_SYMBOL"
+}
+```
+
+In this formula:
+- `UNIT_CODE` is the code to identify a unit
```
Comment (NolwennP) [inline content/concepts/target-market-settings.md]:
```suggestion
- `UNIT_CODE` is the code to identify a unit.
```

File: content/concepts/target-market-settings.md [hunk 12]
```diff
@@ -171,9 +187,106 @@ Below is the JSON standard format representing a measure family.
...
+- `UNIT_LABEL_EN_US` and `UNIT_LABEL_FR_FR` are the labels of a unit in each locale
```
Comment (NolwennP) [inline content/concepts/target-market-settings.md]:
```suggestion
- `UNIT_LABEL_EN_US` and `UNIT_LABEL_FR_FR` are the labels of a unit in each locale.
```

File: content/concepts/target-market-settings.md [hunk 13]
```diff
@@ -171,9 +187,106 @@ Below is the JSON standard format representing a measure family.
...
+          "value": "1"
+        }
+      ],
+      "symbol": "m²"
+    },...
+  ]
+}
+```
+::: info
+You can have at max 100 measurements families and 50 units per measurement family.
```
Comment (NolwennP) [inline content/concepts/target-market-settings.md]:
```suggestion
You can have at max 100 measurements families and 50 units per measurement family.  
```
You need to add a double space at the end of your line to force a going to the next line.

File: content/concepts/target-market-settings.md [hunk 14]
```diff
@@ -171,9 +187,180 @@ Below is the JSON standard format representing a measure family.
...
+
+In this formula:
+- `UNIT_CODE` is the code to identify a unit.
+- `UNIT_LABEL_EN_US` and `UNIT_LABEL_FR_FR` are the labels of a unit in each locale.
+- `CONVERSION_OPERATOR` is the operator for a conversion operation to convert a unit in the standard unit.
+- `CONVERSION_VALUE` is the value for a conversion operation to convert the unit in the standard unit.
+- `UNIT_SYMBOL` is the symbol of the unit.
+
+::: info
+One conversion operation per unit is required and you can have at max 5 conversion operations per unit.
```
Comment (camillebourjade) [inline content/concepts/target-market-settings.md]:
```suggestion
One conversion operation per unit is required and you can have a maximum of 5 conversion operations per unit.
```

File: content/concepts/target-market-settings.md [hunk 15]
```diff
@@ -171,9 +187,180 @@ Below is the JSON standard format representing a measure family.
...
+      "operator": "mul",
+      "value": "1"
+    }
+  ],
+  "symbol": "°K"
+}
+```
+
+
+**With 1 conversion operation for a non standard unit**
```
Comment (camillebourjade) [inline content/concepts/target-market-settings.md]:
```suggestion
**With 1 conversion operation for a non-standard unit**
```

File: content/concepts/target-market-settings.md [hunk 16]
```diff
@@ -171,9 +187,180 @@ Below is the JSON standard format representing a measure family.
...
+    {
+      "operator": "ADD",
+      "value": "273.15"
+    }
+  ],
+  "symbol": "°C"
+}
+```
+
+**With 3 conversion operations for a non standard unit**
```
Comment (camillebourjade) [inline content/concepts/target-market-settings.md]:
```suggestion
**With 3 conversion operations for a non-standard unit**
```

**Pull request merged**